meeteval/viz/visualize.js

```diff
@@ -1489,6 +1491,45 @@ class CanvasPlot {
         }
     }
 
+    function drawRecordingAudioButton(container, rangeSelector, name, recording_file) {
+        let update_audio = function (){
+            // Note: Deleting and adding an audio with the same content doesn't
+            //       trigger a load. Hence, no optimization necessary.
+            audio_div.selectAll("*").remove();
+            let lower = rangeSelector.lower_input.node().value;
+            let upper = rangeSelector.upper_input.node().value;
+            let value = "[" + lower + ":" + upper + "]";
+            let path = server_path.node().value + "/" + file_path.node().value + "::" + value;
```

Comment:
This fails when the file path already contains a slicing operation, doesn't it? I think it would be better to use URL arguments: https://docs.aiohttp.org/en/stable/client_quickstart.html#passing-parameters-in-urls

```
server_url/file.wav?start=0&stop=16000
```

Reply:
Yes, it would fail. I changed it to arguments.
The server accepts now both, the slicing and the arguments.
The js code uses the arguments, but some paths may use the slicing, hence I kept the slicing code.
If both are provided, the code will fail.

Question:
Should both be support double slicing?

e.g. CSS pipeline on UEM data and the user wants to listen to enhanced data and observation.
I am not sure, it that is too special.

Reply:
Ok, I added the support for both arguments. I kept negative slices for the future.